examples/connectivity/plot_connectivity_measures.py

```diff
@@ -0,0 +1,92 @@
+"""
+Comparing different connectivity measures
+=========================================
+
+This example compares different measures of functional connectivity between
```

Comment:
This idea of the example compares different measures. But, the end-result is shown as classification accuracy which is actually overtaken. Will it be good to show example only about measures ? If this is the case, we can also reduce number of ADHD subjects which obviously can save time. Just an idea.